src/Math-Tests-Complex/PMComplexTest.class.st

```diff
@@ -810,6 +822,22 @@ PMComplexTest >> testTwoComplexNumbersWithDifferentRealPartsAreNotEqual [
  	self deny: z equals: w.
 ]
 
+{ #category : #'testing - complex conjugation' }
+PMComplexTest >> testWeCanIdentifyTwoComplexNumbersAsBeingComplexConjugatesOfEachOther [
+	| z |
+	z := 3 - 2 i.
+	
+	self assert: (z isComplexConjugateOf: (3 + 2 i))
```

Comment:
For the same reasons as discussed above, I would name this test `testIsComplexConjugateOf` and I would not create a temporary variable. So the code would be:

```st
testIsComplexConjugateOf

	self assert: ((3 - 2i) isComplexConjugateOf: (3 + 2i)).
```

But this was already in the tests that you propose to remove.

Also, look at those tests that you propose to remove... they cover many important scenario:

- is complex conjugate of complex and real
- is complex conjugate of real and complex (different receiver)
- is complex conjugate of complex and complex
- is complex conjugate of complex and complex when they are not complex conjugate of each other
...

If we keep the `isComplexConjugateOf:` method, we must keep all those tests.
But if we remove it, and add `complexConjugate` to `Number` as I proposed above (will be a separate issue), then we can remove them and simply add `testComplexConjugate` to both `Number` and `PMComplex`